k9mail-library/src/main/java/com/fsck/k9/mail/Message.java

```diff
@@ -92,6 +92,8 @@ public void setInternalDate(Date internalDate) {
 
     public abstract Address[] getRecipients(RecipientType type) throws MessagingException;
 
+    public abstract Address[] getListPost();
```

Comment:
Okay. I'm using getFirstHeader, so I'll probably have to reimplement / refactor that in...